modules/serverless-init-containers-apps.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * /serverless/develop/serverless-applications.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-init-containers-apps_{context}"]
+= Configuring init containers
+
+link:https://kubernetes.io/docs/concepts/workloads/pods/init-containers/[Init containers] are specialized containers that are run before application containers in a pod. They are generally used to implement initialization logic for an application, which may include running setup scripts or downloading required configurations.
+
+[NOTE]
+====
+Init containers may cause longer application start-up times and should be used with caution for serverless applications, which are expected to scale up and down frequently.
+====
+
+Multiple init containers are supported in a single Knative service spec. Knative provides a default, configurable naming template if a template name is not provided. The init containers template can be set by adding an appropriate value in a Knative `Service` object spec.
```

Comment:
At the end of the first sentence, should "spec" be spelled out? I would think so, but maybe it's a special Knative term and not a casual version of "specification."

Reply:
It's referring to the spec in the YAML, but also there is no mention of whether to use "spec" or "specification" in the IBM style guide AFAIK. I'm not sure if it's mentioned in any other guidelines we have, but I think either way for the purposes of Knative, spec probably makes the most sense.